extlinks/organisations/templates/organisations/organisation_charts_include.html

```diff
@@ -139,6 +146,20 @@ <h3>Latest link events</h3>
         },
       },
     });
+
+    // Click all Link Events chart section TESTETETETETETETETET
+    document.getElementById('{{ collection_name }}_linkEvents_button').click();
+    var form_data = {{ form_data|safe }};
+    var collection_id = {{ collection.collection_id|safe }};
```

Comment:
Tiny nit, feel free to ignore: 
Can we use const or let here instead of var?

Reply:
Done!